email_templates/carpeEmail/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
This css /template compiling is server side and is not overwritten. The email html and css are separate from the client side stuff, since the server compiles and populates templates before sending em